### DISCUSSION TITLE
Alert for low prices

### DISCUSSION THREAD

**Discussion by icenov** [amberelectric/public-api#119] [Ideas]
Is it possible to set up an alert for low prices? I can only manually recharge a battery bank and it would be useful if I could get an alert when prices drop below, eg 15 c/kWh (as rare as that may be these days!).

**Comment by madpilot** [comment]
You can with the API, but it requires some plumbing. Do you run a home automation platform?

**Reply by icenov** [reply]
Nothing formal, but I use Node-red with a python script to pull down the prices every 5 mins. I can't think of how to get this back to the phone though - which is where I need the alert.

**Reply by madpilot** [reply]
There are push notification platforms like pusher: https://pusher.com/

We are talking about this as a feature in the app, but it's not on the roadmap at the moment.

**Reply by icenov** [reply]
I currently have an email to sms service with my mobile provider, so might try that. I have used it for some applications and it's quite good, but a bit of mucking about to get the code to securely send an email through gmail etc.
